### PR TITLE
Added a Resistor Color Duo task

### PR DESCRIPTION
**Pull request by annaoliinyk**
https://exercism.org/tracks/python/exercises/resistor-color-duo

Instructions
If you want to build something using a Raspberry Pi, you'll probably use resistors. For this exercise, you need to know two things about them:

Each resistor has a resistance value.
Resistors are small - so small in fact that if you printed the resistance value on them, it would be hard to read.
To get around this problem, manufacturers print color-coded bands onto the resistors to denote their resistance values. Each band has a position and a numeric value.

The first 2 bands of a resistor have a simple encoding scheme: each color maps to a single number. For example, if they printed a brown band (value 1) followed by a green band (value 5), it would translate to the number 15.

In this exercise you are going to create a helpful program so that you don't have to remember the values of the bands. The program will take color names as input and output a two digit number, even if the input is more than two colors!

The band colors are encoded as follows:

Black: 0
Brown: 1
Red: 2
Orange: 3
Yellow: 4
Green: 5
Blue: 6
Violet: 7
Grey: 8
White: 9
From the example above: brown-green should return 15 brown-green-violet should return 15 too, ignoring the third color.

### CODE REVIEW

**Review by AnaKovaGH — CHANGES_REQUESTED** [review]
Please make the dict constant structure outside the function

**Comment by annaoliinyk** [comment]
Fixed

File: resistor_color_duo/resistor_color_duo_task.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Task from https://exercism.org/tracks/python/exercises/resistor-color-duo
+
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+switch = {
```
Comment (AnaKovaGH) [inline resistor_color_duo/resistor_color_duo_task.py]: Constants should be written in upper case

File: resistor_color_duo/resistor_color_duo_task.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                result = switch.get(first_color) * 10 + switch.get(second_color)
+            return result
+        except:
+            logging.error("Invalid color provided")
+    else:
+        logging.error("Invalid input - should be a list")
+
+
+def main():
+    logging.debug(value(["red"]))
```
Comment (AnaKovaGH) [inline resistor_color_duo/resistor_color_duo_task.py]: Change DEBUG to INFO

**Review by AnaKovaGH — CHANGES_REQUESTED** [review]
Please change what was mentioned

**Comment by annaoliinyk** [comment]
Fixed

**Review by AnaKovaGH — APPROVED** [review]

**Pull request merged**